WooCommerce/Classes/ViewRelated/Orders/Simple Payments/Amount/SimplePaymentsAmount.swift

```diff
@@ -48,6 +48,9 @@ final class SimplePaymentsAmountHostingController: UIHostingController<SimplePay
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        // Disables interactive dismiss action.
+        isModalInPresentation = true
```

Comment:
Neat, I have never this property before! However, the idea is to only prevent the dismissal of the view while the network operation is happening, we still should be able to dismiss the view via drag gestures if the user hasn't tapped on the "done" button yet.

Reply:
I will check the possibility.

Reply:
@Ecarrion  I handled in this way: Now do swipe to dismiss if the user hasn't tapped on the "Done" button, otherwise can't.
```
 viewModel.$loading
             .sink { [weak self] loading in
                 // here handling interactive dismissal
                 self?.isModalInPresentation = loading
             }
             .store(in: &subscriptions)
```